### DISCUSSION TITLE
LoRA or LoRA Base?

### DISCUSSION THREAD

**Discussion by StripedPuppy** [LostRuins/koboldcpp#514] [Q&A] [answered]
In the models tab, what is the difference between LoRA and LoRA Base for LoRA models?

I've noticed I can run 7b and 13b LoRA's under LoRA Base, but not LoRA due to insufficient extra RAM.

**Comment by LostRuins** [comment, marked as answer]
lora base is intended to select the original f16 model weights (a higher quality model to use as a base for the layers modified by the adapter.)
